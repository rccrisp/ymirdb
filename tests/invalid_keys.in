set a123456789123456 1
set a! 1
set 1a 1
bye

