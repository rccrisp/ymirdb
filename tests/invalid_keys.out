> key name not permitted

> key name not permitted

> key name not permitted

> bye
